File: Server/ElectricBillPaymentSystem/ModelClass/DTO/UserRole.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ModelClass.DTO
{
    public class UserRole
    {
        public int UserRoleId { get; set; }
        public int RoleId { get; set; }
        public int UserId { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string UpdatedBy { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public int Status { get; set; }
        public virtual Role Role { get; set; }
        public virtual User User { get; set; }
    }
}

File: Client/ElectricityBillManagement-Client/src/app/Model/UserRole.ts
import { Role } from "./Role";
import { User } from "./User";

export class UserRole{
    UserRoleId: number;
    RoleId: number;
    UserId: number;
    CreatedBy: string;
    CreatedDate: Date;
    UpdatedBy: string;
    UpdatedDate: Date;
    Status: number;
    Role: Role;
    User: User;

}